apps/web/modules/email/emails/survey/follow-up.tsx

```
@@ -0,0 +1,21 @@
+}
+
+export function FollowUpEmail({ html, surveyName }: FollowUpEmailProps): React.JSX.Element {
+  return (
+    <EmailTemplate>
+      <Container>
+        <Text className="mb-1">
+          You have receieved a follow up email from the survey: <strong>{surveyName}</strong>
+        </Text>
+      </Container>
+
+      <div dangerouslySetInnerHTML={{ __html: html }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)